io_scene_niftools/kf_import.py

```diff
@@ -81,10 +81,10 @@ def execute(self):
                 # calculate and set frames per second
                 self.transform_anim.set_frames_per_second(kfdata.roots)
                 for kf_root in kfdata.roots:
-                    self.transform_anim.import_kf_root(kf_root, b_armature, bind_data)
+                    self.transform_anim.import_kf_root(kf_root, b_armature)
 
         except NifError:
             return {'CANCELLED'}
 
         NifLog.info("Finished successfully")
-        return {'FINISHED'}
+        return {'FINISHED'}
```

Comment:
I think this is just the removal of a newline at the end of a file, counter to the pep8 style guide.